core/core_bind.cpp

```diff
@@ -519,10 +519,32 @@ double _OS::get_unix_time() const {
 	return OS::get_singleton()->get_unix_time();
 }
 
+/**
+ * Safer method with error checking for use by the scripting API.
+ * This method accepts negative values since the parameter is signed.
+ */
+void _OS::_delay_usec(int p_usec) const {
```

Comment:
Ah well, `_OS::delay_usec()` is already a wrapper so wrapping it further is likely not needed.
I suggested that without having looked at the code, but in C++ you'd use `OS::get_singleton()->delay_usec()` anyway.